evidence/reactor_test.go

```diff
@@ -125,13 +125,13 @@ func sendEvidence(t *testing.T, evpool *Pool, valAddr []byte, n int) types.Evide
 }
 
 var (
-	numEvidence = 10
+	numEvidence = 50
```

Comment:
I changed this to test the maximum throughput (i.e. more as benchmarking) so I can revert this to what it was before